packages/components/src/Divider/Divider.module.scss

```diff
@@ -0,0 +1,41 @@
+@import "~@kaizen/design-tokens/sass/border";
+@import "~@kaizen/design-tokens/sass/color";
+
+// Use different tokens with different opacities across the Zen and Heart themes, by assuming that when Zen is applied there won't be any CSS variables declared.
+
+$dt-content-border-color: rgba($color-gray-600-rgb, 0.1);
+
+.wrapper {
+  width: 100%;
+  border: 0;
+  margin: 0;
+  border-radius: $border-solid-border-radius;
+  // This is here to protect against a global style in a consumer.
+  // https://github.com/cultureamp/murmur/blob/master/app/assets/stylesheets/legacy/modules/_body.scss
+  visibility: visible;
+}
+
+.content {
+  border-top: 1px solid;
+  border-color: $dt-content-border-color;
+
+  &.reversed {
+    border-color: rgba($color-white-rgb, 0.1);
+  }
+}
+
+.canvas {
+  border-top: 1px solid;
+  border-bottom: 1px solid;
+  border-color: rgba($color-gray-600-rgb, 0.1);
+
+  &.reversed {
+    border-color: rgba($color-white-rgb, 0.1);
+  }
+}
+
+.menuSeparator {
+  @extend .content; /* stylelint-disable-line scss/at-extend-no-missing-placeholder */
```

Comment:
Apparently the solution to this would be to make something like this
```
// Silent class - maybe give a better name?
%content { ... }

.content {
  @extend %content;
}

.menuSeparator {
  @extend %content;
  margin: 5px 0;
}
```

Reply:
Not here to solve SCSS issues